src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.WriteTests.cs

```diff
@@ -139,7 +139,7 @@ private async Task WriteAsync(TestStream stream)
             }
 
             // We don't auto-flush.
-            Assert.Equal(0, stream.TestFlushCount);
+            //Assert.Equal(0, stream.TestFlushCount);
```

Comment:
Commented out code.

Reply:
Not sure how valuable this check is, so leaning towards deleting it.

For PipeWriter tests we do end up calling the underlying Stream.FlushAsync https://source.dot.net/#System.IO.Pipelines/System/IO/Pipelines/StreamPipeWriter.cs,343

The Stream Serializer overload goes through PooledByteBufferWriter and calls Stream.WriteAsync https://source.dot.net/#System.Text.Json/src/libraries/Common/src/System/Text/Json/PooledByteBufferWriter.cs,79

Reply:
Then we can delete it.